src/overlays/actors/ovl_Obj_Ocarinalift/z_obj_ocarinalift.c

```diff
@@ -64,8 +64,7 @@ void ObjOcarinalift_Init(Actor* thisx, PlayState* play) {
     this->dyna.actor.world.rot.z = 0;
     DynaPolyActor_Init(&this->dyna, 1);
     DynaPolyActor_LoadMesh(play, &this->dyna, &object_raillift_Colheader_0048D0);
-    this->unk160 = this->dyna.actor.home.rot.z * 0.1f;
-    if (1) {}
+    this->unk160 = thisx->home.rot.z * 0.1f;
```

Comment:
If you use thisx in one place, and it can be used for all the other actor accesses, please do so.

Reply:
i couldn't get all the macros to take thisx for some reason